static/templates/draft.hbs

```diff
@@ -33,7 +33,7 @@
                 <div class="messagebox-content">
                     <div class="message_top_line">
                         <div class="draft_controls">
-                            <i class="fa fa-pencil fa-lg restore-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Restore draft' }}"></i>
+                            <i class="fa fa-pencil fa-lg restore-draft tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Restore draft' }} (Enter)"></i>
```

Comment:
Can you move this to a new PR?  I ask mainly because the popover looks too narrow and thus probably needs some styling tweak.

Reply:
#18819